.readme-partials.yml

```diff
@@ -0,0 +1,27 @@
+body: |-
+        ### Using TypeScript
+
+        `@google-cloud/cloudbuild` provides TypeScript type definitions, and can
+        be used inside a TypeScript project:
+
+        ```js
```

Comment:
@alexander-fenster I was thinking it would be nice if we could basically generate a blurb like this.

Reply:
I think we could keep this as simple as "We include TypeScript types".  Most TypeScript users are going to be able to figure out what to do from there :)

Reply:
We won't do it in Beta. We use [nunjucks](https://www.npmjs.com/package/nunjucks) templates for generating stuff so adding any regular file should be pretty straightforward but for beta, I suggest we concentrate on functionality (e.g. moving existing libraries to TS), and files needed for new libraries will be taken care of after that.